Android/app/src/main/java/io/github/project_travel_mate/destinations/description/FinalCityInfoPresenter.java

```diff
@@ -18,8 +18,8 @@
  */
 
 class FinalCityInfoPresenter {
-    private FinalCityInfoView mFinalCityInfoView;
-    private final OkHttpClient mOkHttpClient;
+    private FinalCityInfoView   mFinalCityInfoView;
```

Comment:
@Swati4star a rule for ensuring no double spaces can be added in checkstyle

Reply:
Sure. I'll look into the checkstyle rules.